examples/mobile/SchedulingApp/create-event.html

```diff
@@ -17,12 +17,83 @@
 				<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
 			</div>
 			<div class="ui-appbar-title-container">
-				<span class="ui-appbar-title">Turn device on</span>
+				<span class="ui-appbar-title">Turn device on and off</span>
 			</div>
 		</header>
 		<div class="ui-content">
-			
+			<div style="font-size: 15px; margin: 0 0 15px 24px">Max running time is 24 hours.
+				<br>Please set time less than 24 hours.
+			</div>
+			<div class="ui-content-subheader">Device on</div>
```

Comment:
Mixed coding style

`<div class="ui-content-subheader">Device on</div>`

vs

```
<div class="ui-content-subheader">
    Device on
</div>
```
I think it would be better to use common approach in this file (or app).

Reply:
fixed